ui-configuration.graphqls

```diff
@@ -23,11 +23,26 @@ type DashboardConfiguration {
     configuration: String!
 }
 
+type MenuItem {
+    # Display name
+    name: String!
+    # Showing icon
```

Comment:
Is this icon name?

Reply:
Yes, I have updated the description to the icon name.